packages/data-portal-schema/src/lib/dataSchemaHelpers.ts

```diff
@@ -576,3 +582,54 @@ function normalizeEntity(
 ): (string | BaseEntity)[] {
     return !entity ? [] : !Array.isArray(entity) ? Array(entity) : entity;
 }
+
+export function findConditionalAttributes(
+    schemaData: DataSchemaData,
+    dataSchemaMap: { [id: string]: DataSchemaData }
+): string[] {
+    const conditionalAttributes: string[] = [];
+
+    for (const [, value] of Object.entries(dataSchemaMap)) {
```

Comment:
Can't we use `Object.values()` or `_.values()` instead?

```suggestion
    Object.values(dataSchemaMap).forEach(value => {
```